src/organisms/attend-button.tsx

```diff
@@ -34,6 +34,11 @@ export default function AttendButton({ state, onClick }: AttendButtonProps) {
       btnClass: "overtime",
       icon: <RxCross1 className={aButton.icon} />,
     },
+    DISABLED: {
+      label: "出席申請中",
```

Comment:
```suggestion
      label: "…",
```
この状態はページの読み込み中やログイン情報の取得中にも使われますので, 単純に三点リーダーのほうがよいかと思います.